spec.md

```diff
@@ -914,10 +919,16 @@ reg myreg: SInt, myclock
 ```
 
 Optionally, for the purposes of circuit initialization, a register can be
-declared with a reset signal and value. In the following example,
-`myreg`{.firrtl} is assigned the value `myinit`{.firrtl} when the signal
-`myreset`{.firrtl} is high.
-
+declared with a reset signal and value. The register's value is updated with the
```

Comment:
I'd also strike the "Optionally, for the purposes of circuit initialization" (since you're here).

Reading this paragraph in its entirety is somewhat confusing because it mentions both a possible user intended function, "circuit initialization", when introducing a semantic, "register reset". This is confusing because "register initialization" at power on (which could easily be confused with "circuit initialization") is a separate, critical concept that isn't addressed by the spec.